validation/start-block_test.go

```diff
@@ -0,0 +1,79 @@
+package validation
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/internal/bindings"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func testStartBlock(t *testing.T, chain *ChainConfig) {
+	rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+	require.NotEmpty(t, rpcEndpoint)
+
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	systemConfigAddress := Addresses[chain.ChainID].SystemConfigProxy
+	require.NotZero(t, systemConfigAddress)
+
+	offChainParam := chain.Genesis.L1.Number
+	onChainParam, err := getStartBlockWithRetries(context.Background(), common.Address(systemConfigAddress), client)
+	require.NoError(t, err)
+
+	if offChainParam > onChainParam {
+		// Ensure there aren't any skipped deposits in the block gap
+		portalAddress := Addresses[chain.ChainID].OptimismPortalProxy
+		require.NotZero(t, portalAddress)
+
+		missedEvents, err := checkForDepositEvents(client, common.Address(portalAddress), onChainParam, offChainParam)
+		require.NoError(t, err)
+		require.Zero(t, len(missedEvents))
```

Comment:
```suggestion
		require.Len(t, missedEvents, 0)
```
and note that the `len` of a `nil` slice is defined as `0`.